open-machine-learning-jupyter-book/deep-learning/image-segmentation.ipynb

```diff
@@ -0,0 +1,1833 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    
src should be <code>https://static-1300131294.cos.ap-shanghai.myqcloud.com/html/image-segmentation/index.html</code>
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/666/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>